package/MDAnalysis/core/universe.py

```diff
@@ -1169,7 +1204,7 @@ def add_dihedrals(self, values, types=None, guessed=False):
         .. versionadded:: 1.0.0
         """
         self._add_topology_objects('dihedrals', values, types=types,
-                                 guessed=guessed)
+                                   guessed=guessed)
```

Comment:
```suggestion
                                   guessed=guessed)
```

Revert unrelated change.